contrib/curry/podspec/hostpath-media/kustomization.yaml

```
@@ -2,7 +2,9 @@ apiVersion: kustomize.config.k8s.io/v1alpha1
 kind: Component
 patches:
   - target:
-      labelSelector: app.kubernetes.io/managed-by=kustomize, app.kubernetes.io/component=application
+      labelSelector: >-
+        app.kubernetes.io/managed-by=kustomize,
+        app.kubernetes.io/component=application
       annotationSelector: kustomize/patch-target=pod-spec
```

I know that it is only a line, a single annotation, but for consistency. It is a NIT tho.
```suggestion
      annotationSelector: >-
        kustomize/patch-target=pod-spec
```